tests/contrib/flask/test_flask_appsec.py

```diff
@@ -397,7 +395,7 @@ def test_sqli(param_str):
                 "valueParts": [{"value": "SELECT 1 FROM "}, {"value": "sqlite_master", "source": 0}]
             }
             assert loaded["vulnerabilities"][0]["location"]["path"] == "tests/contrib/flask/test_flask_appsec.py"
-            assert loaded["vulnerabilities"][0]["location"]["line"] == 370
+            assert loaded["vulnerabilities"][0]["location"]["line"] == 368
```

Comment:
Do we know why this changed?

Reply:
Whenever the line count in this file changes, these expectations also change.